ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/KeyValueParticipantStateWriter.scala

```diff
@@ -31,7 +31,7 @@ class KeyValueParticipantStateWriter(writer: LedgerWriter, metrics: Metrics) ext
         transactionMeta,
         transaction.assertNoRelCid(cid => s"Unexpected relative contract id: $cid"),
       )
-    val correlationId = nextSubmissionId()
+    val correlationId = submitterInfo.commandId
```

Comment:
Perhaps inline `correlationId` and just name the parameter?

Reply:
True, that's even simpler -- changed.